tests/API/OVAL/skip_paths/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+add_oscap_test("test_skip_paths.sh")
```

Comment:
I'm not insisting on it, but could it make more sense to have it in `/tests/probe_behavior/`? Just FYI.